articles/quickstart/native/android/06-calling-apis.md

```diff
@@ -78,4 +88,6 @@ client.newCall(request).enqueue(new Callback() {
 });
 ```
 
-From here, check that the request was made and that the response came back as expected. You will need to configure your server-side to protect your API endpoints with the secret key for our Auth0 application. As in this example we're using the Auth0's issued `access_token`, you can use this same token to call Auth0 API's.
+Check if the request was made and if the response that came back was what you expected. 
+
+You need to configure your server side to protect your API endpoints with the secret key for our Auth0 application. In this example, you can use the user's access token issued by Auth0 to call Auth0 APIs.
```

Comment:
I think we refer to "Auth0 applications" as "Auth0 clients" previously. Also, I would change: our Auth0 -> your Auth0